src/sources/internal_logs.rs

```diff
@@ -64,26 +65,26 @@ async fn run(
         .chain(tokio_stream::wrappers::BroadcastStream::new(
             subscription.receiver,
         ))
+        .filter_map(|log| future::ready(log.ok()))
```

Comment:
instead of having the match, the `BroadcastStreamRecvError` that were ignored are now filtered before reading the stream.